students/jinee_han/lesson03/Mail.py

```diff
@@ -0,0 +1,126 @@
+
+# --------------------------------
+# 07/02/19 Jinee Han
+# Python Programming Lesson 3
+# Mailroom Part 1
+# ---------------------------------
+
+
+'''
+Program Flow
+1. Donor list is saved as a 'Donor_List.txt' file.
+2. Display an options list to the user.
+ : List 1: Send a Thank you note, 2: Create a List, 3: Exit
+# When the user select 1,
+a) Ask the user the donor name.
+If typed 'list', display the donor name list.
+If typed donor's name not listed, add the data to the Donor_List.
+If typed donor's name listed, use the name.
+b) Ask the user the amount.
+Add the amount to the Donor_List.
+c) create a thank you email and print to terminal.
+d) Return to the original prompt.
+'''
+
+import sys
+
+# Donor List
+donor_db = [["Kim Kardasian", [653772.32, 12.17]],
+            ["Kendal Jenner", [877.33]],
+            ["Gigi Hadid", [663.23, 43.87, 1.32]],
+            ["Justin Bieber", [1663.23, 4300.87, 10432.0]],
+            ["Will Smith",[43.23, 4000.07, 183423.2]]
+            ]
+
+# Display options
+prompt = "\n".join(("Welcome to the donor list!",
+          "Please choose from below options:",
+          "1 - Send a Thank you",
+          "2 - Create a Report",
+          "3 - Exit",
+          ">>> "))
+
+def send_thank_you_note():
+    inputValue = input("Enter a full name. (Type 'list' to see the donor list)")  # Ask the name
```

Comment:
variable names follow snake_case convention -> `input_value`

Reply:
Thank you. I will do that in the future.